modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -0,0 +1,313 @@
+include::mule-runtime::partial$mmp-concept.adoc[tag=serverParameterDescription]
+include::mule-runtime::partial$mmp-concept.adoc[tag=propertiesParameterDescription]
+include::mule-runtime::partial$mmp-concept.adoc[tag=skipParameterDescription]
+include::mule-runtime::partial$mmp-concept.adoc[tag=skipDeploymentVerification]
+include::mule-runtime::partial$mmp-concept.adoc[tag=authTokenParameterDescription]
+include::mule-runtime::partial$mmp-concept.adoc[tag=connectedAppsParameterDescription]
+| `deploymentSettings` | Any of the parameters documented in <<deployment-settings, deploymentSettings Reference>>
+| No
+|===
+// end::DeploymentReference[]
+
+// tag::DeploymentSettingsReference[]
+[[deployment-settings]]
+=== deploymentSettings Parameters Reference
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+|===
+|Parameter | Description
+| `enforceDeployingReplicasAcrossNodes` | Enforces the deployment of replicas across different nodes. The default value is `false`.
+
+Configuration example:
+[source,xml,linenums]
+----
+<deploymentSettings>
+  <enforceDeployingReplicasAcrossNodes>false</enforceDeployingReplicasAcrossNodes>
+</deploymentSettings>
+----
+| `updateStrategy` |
+[cols="1,3"]
+!===
+! Accepted values ! Description
+
+! `rolling` ! Maintains availability by updating replicas incrementally. Requires one additional replica’s worth of resources to succeed. +
+If `enforceDeployingReplicasAcrossNodes` is enabled, the maximum number of replicas you can configure is one less than the total number of nodes.
+! `recreate` ! Terminates replicas before re-deployment. Re-deployment is quicker than `rolling` and doesn’t require additional resources. +
+If `enforceDeployingReplicasAcrossNodes` is enabled, the maximum number of replicas you can configure is equal to the number of nodes.
+!===
+
+The default value is `rolling`.
+
+Configuration example:
+[source,xml,linenums]
+----
+<deploymentSettings>
+  <updateStrategy>recreate</updateStrategy>
+</deploymentSettings>
+----
+| `forwardSslSession` | Enables SSL forwarding during a session. The default value is `false`.
+
+Configuration example:
+[source,xml,linenums]
+----
+<deploymentSettings>
+  <forwardSslSession>true</forwardSslSession>
+</deploymentSettings>
+----
+| `clustered` | Enables clustering across two or more replicas of the application. The default value is `false`. +
+
+Configuration example:
+[source,xml,linenums]
+----
+<deploymentSettings>
+    <clustered>true</clustered>
+</deploymentSettings>
+----
+| `lastMileSecurity` | Enable Last-Mile security to forward HTTPS connections to be decrypted by this application +
+This requires an SSL certificate to be included in the Mule application, and also requires more CPU resources. The default value is `false`. +
+Configuration example:
+[source,xml,linenums]
+----
+<deploymentSettings>
+    <lastMileSecurity>true</lastMileSecurity>
+</deploymentSettings>
+----
+| `resources` |
```

Comment:
We should remove this, resources is not used for this deploy. In cloudhub 2 users should use Vcores to allocate resources